sdk/resourcemanagerhybrid/README.md

```diff
@@ -131,7 +129,52 @@ AzureResourceManager azure = AzureResourceManager
     .withDefaultSubscription();
```

Comment:
You probably want to replace this example, as it only works on public cloud.

Reply:
Updated